Unix/build.mak

```diff
@@ -102,7 +102,7 @@ endif
 ifdef CONFIG_SINGLEIMAGE
   DIRECTORIES += image
 else
-  DIRECTORIES += server agent
+  DIRECTORIES += server agent engine
```

Comment:
So it looks like there's a new binary here, but it hasn't been put into the kit. This means that the RPM and DPKG kits that we build won't be valid, and will install a non-working version of OMI.

Note that PBUILD currently only builds --dev, not kits. Building kits are done on a restricted set of machines (just SLES 10 for Linux) due to universal binaries. You minimally need to make the `installbuilder` changes, build new kits on a SLES 10 system, and then test it to make sure that the version you install actually works.

I'm going to need this to be done prior to signing off this change.